docs/README.md

```diff
@@ -0,0 +1,28 @@
+## Quick Start to Docs
```

Comment:
This documentation seems somewhat redundant relative to [the main README.md section](https://github.com/openstax/output-producer-service/tree/bn/cnx%23929/production-ready-details#view-the-docs). Should the improvements be made there, or perhaps that be moved here?

Also, there's a note there about documentation automatically rebuilding. If this isn't working as intended, perhaps there's a bug? (I see we mount files as a volume and also copy files in the `Dockerfile` -- maybe that's causing an issue).

Reply:
To note: The documentation does rebuild for the most part, but sometimes the navigation changes don't take when updating the index.rst/ the toc tree.

Reply:
You have to restart the entire container if you change navigational pieces.